src/main/java/com/maxmind/geoip2/model/AnonymousIpResponse.java

```diff
@@ -91,6 +107,15 @@ public boolean isPublicProxy() {
         return isPublicProxy;
     }
 
+    /**
+     * @return This is true if the IP address is on a suspected anonymizing
```

Comment:
It looks like all of the other boolean docs are of the form `@return whether ...`. Perhaps we should update this to be similar here and in the Traits class.

Reply:
I updated it here. For the Traits class, most of them were in this form. I updated one that wasn't. I can make the Traits class use the whether form if you think it's best too though.

Reply:
Weird. I don't think it matters too much. Ideally we would be consistent everywhere, but this seems good enough given the current state of things.